docs/reference/lambda-support.md

```diff
@@ -0,0 +1,435 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/python/current/lambda-support.html
+---
+
+# Monitoring AWS Lambda Python Functions [lambda-support]
+
+The Python APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
+
+```
+Note: The Centralized Agent Configuration on the Elasticsearch APM currently does NOT support AWS Lambda.
+```
+
+
+## Prerequisites [_prerequisites]
+
+You need an APM Server to send APM data to. Follow the [APM Quick start](docs-content://solutions/observability/apps/get-started-with-apm.md) if you have not set one up yet. For the best-possible performance, we recommend setting up APM on {{ecloud}} in the same AWS region as your AWS Lambda functions.
+
+
+## Step 1: Select the AWS Region and Architecture [_step_1_select_the_aws_region_and_architecture]
+
+<style>
```

Comment:
Rendering of this is broken

Reply:
I'll take a closer look at this today.

Reply:
elastic/docs-builder does not support CSS and JavaScript in Markdown, and there is no plan to support it in the future. I made some updates in https://github.com/elastic/apm-agent-python/pull/2221/commits/e795df5d8275c4a7399b24546e83212c3c99a934. Take a look at [the preview](https://docs-v3-preview.elastic.dev/elastic/apm-agent-python/pull/2221/reference/lambda-support#add_the_apm_layers_to_your_lambda_function) and let me know what you think about this approach.